index.md

```diff
@@ -257,4 +257,4 @@ In the IPSJ Journal of Programming (PRO), 2005. *In Japanese*.
 
 ## Standards
 
-<a class="paper" name="iso" href="#iso">#</a> IPA Ruby Standardization WG. **[Programming Languages — Ruby (Draft)](https://www.ipa.go.jp/osc/english/ruby/)**. Final draft of the Ruby ISO standard, 2010.
+<a class="paper" name="iso" href="#iso">#</a> IPA Ruby Standardization WG. **[Programming Languages — Ruby (Draft)](https://web.archive.org/web/20190401051050/https://www.ipa.go.jp/osc/english/ruby/)**. Final draft of the Ruby ISO standard, 2010.
```

Comment:
This worries me - I'll ask around about where it is now.

Reply:
https://twitter.com/ChrisGSeaton/status/1222192770871369728

Reply:
It looks like this has been deliberately removed by the authoring body. Since they now only present a paid option, I'm not sure it's right to link to the archived old free version. What do you think?

Reply:
As Aaron translated "they don't have it anymore". Difficult to say, but I'd rather delegate the legal questions to Web Archive and change the bibliography link to point there.

If somebody wants some information to disappear the way is to request search engines and archives to delete them. Web Archive deletes data upon request:

```
How can I exclude or remove my site's pages from the Wayback Machine?

You can send an email request for us to review to info@archive.org with the URL (web address) in the text of your message.
```

Reply:
Hmmm thanks very much for taking the time to talk me through your reasoning but I'm afraid I don't agree and I'd rather not use archive.org to link to a paper that the original authors have taken down for whatever reason. Could we merge the rest of the PR without that?

Reply:
Not a problem :)
Do you want to preserve the bib item without any links?

Reply:
Yes should still link to it - it's not a link of freely available research. We'll probably find more papers in the future that don't have a public version available (we've been lucky so far.) Could link to the paid version, or not link to anything. I think I'd do the former.

Reply:
Shall we merge this, then?